lib/jose-jwe-decrypt.js

```diff
@@ -22,9 +22,10 @@
  * @param key_promise    Promise<CryptoKey>, either RSA or shared key
  */
 JoseJWE.Decrypter = function(cryptographer, key_promise) {
-  this.cryptographer = cryptographer;
-  this.key_promise = key_promise;
-  this.headers = {};
+  var that = this;
```

Comment:
I feel it would be better to improve the minifier to replace `this` with a variable instead of making the code harder to read. The whole thing is only worth doing if `this.foo` appears 5 or more times in the body of the function (i.e. you are making the code longer, not shorter in this specific case).